rfcs/quic.md

```diff
@@ -0,0 +1,104 @@
+# RFC 42: Add an optional QUIC server to wpt
+
+## Summary
+
+Add an **optional** [QUIC](https://quicwg.org/) server to wpt, implemented with
+[aioquic](https://github.com/aiortc/aioquic) in Python 3 and listening on port
+`9001` by default. This server is used to test web APIs that rely on the
+QUIC protocol (e.g. [WebTransport](https://wicg.github.io/web-transport/)). The
+server is optional but tries to start by default; `wpt` front-end will skip the
```

Comment:
Can we make this opt-in initially, like we did for http2? In fact I think I'd prefer "opt in and fails if you opt in and it can't start" rather than ending up in a situation where you only realise the server failed to start when investigating why tests aren't running.

Reply:
Sure we can do that. I'll change the RFC later.